examples/autoquality/autoquality_usage.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>What exactly is the dataset? Any sources to cite?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Toloka/toloka-kit/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>